web/src/recent_topics_ui.js

```diff
@@ -415,10 +416,11 @@ function format_conversation(conversation_data) {
             )
             .map((user) =>
                 render_user_with_status_icon({
-                    name: user.full_name,
+                    name: people.get_display_full_name(user.id),
```

Comment:
Could you split this out into a separate commit? This seems not related to the main bug fix.

Reply:
Oh! I see this is for showing "Muted User" in huddles.
Even in that case, I think this would be better in it's own commit, with an appropriate commit message :)